3684/0/about_classes.rb

```diff
@@ -10,169 +14,176 @@ def test_instances_of_classes_can_be_created_with_new
   end
 
   # ------------------------------------------------------------------
-
-  class Dog2
-    def name(a_name)
-      @attr_name = a_name
+  # class about dog number 2
+  class DogNumberTwo
```

Comment:
so. correct me if I wrong. Smbd with the id 3501 completed this task. After that, you've decided to base your changes on the work of 3501. If so - I'm not sure that this approach is acceptable.

Reply:
No, I forked rep when 3501 was merged to master, then in taskbranch I started work

Reply:
work on what? On the refactoring of the 3501's work?

Reply:
when i forked my koans was already done, then i start trying to pass hound, this mathes in code can appear when i did this, because i coudn't find solution on the internet, i find them in already merged PR, but i didn't copy whole task from other rep.